src/openMVG/sfm/sfm_data_filters_frustum.cpp

```diff
@@ -86,9 +90,16 @@ Pair_Set Frustum_Filter::getFrustumIntersectionPairs() const
 #endif
   for (int i = 0; i < (int)viewIds.size(); ++i)
   {
+    // Prepare vector of intersecting objects (within loop to keep it
+    // thread-safe)
+    std::vector<HalfPlaneObject> objects = bounding_volume;
+    objects.insert(objects.end(),
```

Comment:
why not consider two insert. inserting an empty object ad modifying it later on is a bit dangerous.
Else consider to user (*objects.back()) =   frustum_perView.at(viewIds[j]);
We can discuss on it.